app/src/main/res/values/app.xml

```diff
@@ -4,6 +4,8 @@
    - file, You can obtain one at http://mozilla.org/MPL/2.0/. -->
 <resources>
     <string name="app_name" translatable="false">Firefox Focus</string>
+    <string name="beta_app_name" translatable="false">Firefox Focus Beta</string>
```

Comment:
We have build variants for strings so we could put these in the correct directories and the Android OS would know how to pick the right `app_name`:

| Directory | `app_name` in the strings.xml | Notes |
|----------|--------------------------------|-------|
| `/main` | `Firefox Focus` | Default string for all variants (including debug) |
| `/focusBeta` | `Firefox Focus Beta` | |
| `/focusNightly` | `Firefox Focus Nightly` | |
| `/klar` | `Firefox Klar` | Default string for all Klar variants (although we don't ship nightly/beta for it) |

Reply:
Ah, in Focus it looks like we put the app name in the `app.xml` instead of the `strings.xml` probably so that we don't mistakenly try to translate it.